pkg/state/state_ofed.go

```diff
@@ -783,6 +825,40 @@ func (s *stateOFED) handleCertConfig(
 	return nil
 }
 
+// handleSubscriptionVolumes handles additional mounts required for subscriptions
+func (s *stateOFED) handleSubscriptionVolumes(
+	ctx context.Context, osname string, runtime string, mounts *additionalVolumeMounts) error {
+	reqLogger := log.FromContext(ctx)
+	if osname == "rhel" && runtime != nodeinfo.CRIO {
```

Comment:
do we want to ignore an empty runtime value ?

Reply:
Can an empty runtime happen?

Reply:
field should be set. just wondering if we should add check for completness.